src/libs/ajax/leonardo/Disks.ts

```diff
@@ -13,12 +13,13 @@ export const Disks = (signal) => {
   });
 
   const v1Func = () => ({
-    list: async (labels = {}): Promise<ListDiskItem[]> => {
+    list: async (labels = {}): Promise<PersistentDisk[]> => {
       const res = await fetchLeo(
         `api/google/v1/disks${qs.stringify(labels, { addQueryPrefix: true })}`,
         _.mergeAll([authOpts(), appIdentifier, { signal }])
       );
-      return res.json();
+      const disks: ListDiskItem[] = await res.json();
+      return mapToPdTypes(disks);
```

Comment:
so this `mapToPdTypes` function takes in `disks: RawPersistentDisk[]` from what I see in disk-utils, but this is saying that disks is a `ListDiskItem[]` are those two types equivalent?

Reply:
wait I just scrolled further and saw that they are, nvmd